survey_mailer/survey_mailer.go

```diff
@@ -0,0 +1,204 @@
+package survey_mailer
+
+import (
+	"log"
+	"strings"
+	"time"
+
+	"github.com/dxe/adb/config"
+	"github.com/dxe/adb/model"
+	"github.com/jmoiron/sqlx"
+	"github.com/sourcegraph/go-ses"
+)
+
+func sendMissingEmail(eventName string, attendees []string) {
+	if len(attendees) > 0 {
+		to := config.SurveyMissingEmail
+		subject := "Missing emails for survey: " + eventName
+		bodyText := "The following people did not receive a survey for this protest due to not having a valid email address: "
+		bodyText += strings.Join(attendees, ", ")
+		bodyHtml := bodyText
+		sendEmail(to, subject, bodyText, bodyHtml)
+	}
+}
+
+func sendEmail(to string, subject string, bodyText string, bodyHtml string) {
+	from := config.SurveyFromEmail
+	bodyHtml += `<br /><img src="https://adb.dxe.io/static/img/logo1.png" height="46" width="50">`
+	// EnvConfig uses the AWS credentials in the environment
+	// variables $AWS_ACCESS_KEY_ID nd $AWS_SECRET_KEY.
+	_, err := ses.EnvConfig.SendEmailHTML(from, to, subject, bodyText, bodyHtml)
+	if err != nil {
+		log.Printf("Error sending email: %s\n", err)
```

Comment:
I'd return the error so that the caller can handle it more intelligently. E.g., bulkSendEmails could not which send attempts fail, and include it in the `sendMissingEmail` alert; while `sendMissingEmail` could report it to the log.

Reply:
Oo... Good idea. Will try that.

Reply:
Please review latest commit. Not sure if I did it the best way or not. It's my first time trying something like this in go.